schemas/modelValidation.schema.tpl.json

```diff
@@ -0,0 +1,30 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/ModelValidation",
```

Comment:
where is this activity used? how does this connect to the validation test (version) schemas?

Reply:
It is used in workflows. The connection is in the "input" property, which includes ValidationTestVersion.